instrumentation/elasticsearch/elasticsearch-transport-5.3/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/elasticsearch/transport/v5_3/springdata/Elasticsearch53SpringTemplateTest.java

```diff
@@ -402,13 +346,128 @@ void resultsExtractor() {
                     span.hasName("SearchAction")
                         .hasKind(SpanKind.CLIENT)
                         .hasNoParent()
-                        .hasAttributesSatisfyingExactly(
-                            equalTo(
-                                maybeStable(DB_SYSTEM),
-                                DbIncubatingAttributes.DbSystemIncubatingValues.ELASTICSEARCH),
-                            equalTo(maybeStable(DB_OPERATION), "SearchAction"),
-                            equalTo(stringKey("elasticsearch.action"), "SearchAction"),
-                            equalTo(stringKey("elasticsearch.request"), "SearchRequest"),
-                            equalTo(stringKey("elasticsearch.request.indices"), indexName))));
+                        .hasAttributesSatisfyingExactly(searchActionAttributes(indexName))));
+  }
+
+  private static List<AttributeAssertion> refreshActionAttributes(String indexName) {
+    List<AttributeAssertion> assertions =
+        new ArrayList<>(
+            asList(
+                equalTo(
+                    maybeStable(DB_SYSTEM),
+                    DbIncubatingAttributes.DbSystemIncubatingValues.ELASTICSEARCH),
+                equalTo(maybeStable(DB_OPERATION), "RefreshAction")));
+
+    if (Boolean.getBoolean("testExperimental")) {
```

Comment:
if you added `testExperimental` only to tell whether tests are run with experimental flag then you could have used `otel.instrumentation.elasticsearch.experimental-span-attributes` too as it is also a system property. Keeping `testExperimental` is fine too.